src/command_modules/azure-cli-profile/azure/cli/command_modules/profile/custom.py

```diff
@@ -91,6 +91,14 @@ def login(username=None, password=None, service_principal=None, tenant=None,
 
     profile = Profile()
 
+    if os.environ.get('ACC_CLOUD', None):
+        console_tokens = os.environ.get('AZURE_CONSOLE_TOKENS', None)
+        if console_tokens:
+            return profile.find_subscriptions_in_cloud_console(re.split(';|,', console_tokens))
+        else:
+            raise CLIError("Executing the 'login' command inside the Azure Cloud Console is unnecessary as"
```

Comment:
Update error string:

"Azure Cloud Shell relies on the user account it was initially launched under, as a result az login is disabled."